Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
 COPY ./ /root/HElib/
 
 RUN apt update && \
     apt install -y build-essential wget git cmake m4 libgmp-dev file && \
     cd ~ && \
-    wget https://www.shoup.net/ntl/ntl-11.4.1.tar.gz && \
-    tar --no-same-owner -xf ntl-11.4.1.tar.gz && \
-    cd ntl-11.4.1/src && \
+    wget https://libntl.org/ntl-11.5.1.tar.gz && \
```

Comment:
On Ubuntu 20.04 you don't need to download from source. You can do `apt-get install libntl-dev`.

Reply:
support for latest NTL is not always available via apt.

Reply:
On ubunt 20.04, "apt"  installs 11.4.3 and not 11.5.1 !

Reply:
Fair enough. We should still ideally be testing with the "default" version that comes with apt.